Task 3/DFS

```diff
@@ -0,0 +1,85 @@
+##Depth First Search (DFS) Java Program
```

Comment:
This file should be under a folder names DFS.

Something like `Task 3/DFS/DFS.java`

Reply:
Hey, I tried making changes but it's showing  "Sorry, a file exists where you’re trying to create a subdirectory. Choose a new path and try again". Can you please help me out

Reply:
Let me have a look!